packages/atomic/src/components/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -10,15 +10,14 @@ import {
   CategoryFacetSortCriterion,
 } from '@coveo/headless';
 import {Initialization} from '../../utils/initialization-utils';
-import {randomID} from '../../utils/utils';
 
 @Component({
   tag: 'atomic-category-facet',
   styleUrl: 'atomic-category-facet.css',
   shadow: true,
 })
 export class AtomicCategoryFacet {
-  @Prop() facetId = randomID('categoryFacet');
+  @Prop({mutable: true}) facetId = '';
```

Comment:
I was able to mutate the value without setting this option to `true`, even though the docs say [props are immutable by default](https://stenciljs.com/docs/properties#prop-decorator). I added the option anyway in case this is a bug waiting to be patched.